### PR TITLE
updated dependencie and enable to edit consumer name

### PR DESCRIPTION
**Pull request by shubhamdevgupta**

### CODE REVIEW

File: frontend/mgramseva/pubspec.yaml [hunk 1]
```diff
@@ -41,7 +44,7 @@ dependencies:
   flutter_spinkit: ^5.0.0
   json_annotation: ^4.8.0
   url_strategy: ^0.2.0
-  screenshot: ^2.1.0
+  screenshot: ^3.0.0
```
Comment (Hari-egov) [inline frontend/mgramseva/pubspec.yaml:47]: screenshot: ^3.0.0 had breaking changes,
 which might cause issue in Print Mini  Reciept ,
verify the functionality once with 3.0.0 then only merge ,
else you can go with lower supported versions.

**Review by Hari-egov — CHANGES_REQUESTED** [review]
screenshot: ^3.0.0 had breaking changes,
which might cause issue in Print Mini Reciept ,
verify the functionality once with 3.0.0 then only merge ,
else you can go with lower supported versions.

File: frontend/mgramseva/pubspec.yaml (hunk 1 above)
Comment (shubhamdevgupta) [reply frontend/mgramseva/pubspec.yaml:47]: I'm using latest dart sdk so it was not supported the lower version of screenshot , and we have not mini reciept printer so, you can check at your end let me know if it's works or not

Comment (shubhamdevgupta) [reply frontend/mgramseva/pubspec.yaml]: @Hari-egov  i have downgrade the screenshot version now , @debasishchakraborty-egovt  merge the pr

**Review by pradeepkumarcm-egov — APPROVED** [review]

**Review by pradeepkumarcm-egov — APPROVED** [review]

**Pull request merged**